src/stories/components/MainMenu/MainMenu.js

```diff
@@ -0,0 +1,122 @@
+import './styles.scss';
+
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  NavLink,
+} from 'react-router-dom';
+import { ReactComponent as DashboardIcon} from
+  '../../../images/icon-dashboard.svg';
+import { ReactComponent as ExperimentsIcon} from
+  '../../../images/icon-experiments.svg';
+import { ReactComponent as AccountIcon} from
+  '../../../images/icon-account.svg';
+import { ReactComponent as LogoutIcon} from
+  '../../../images/icon-logout.svg';
+
+/**
+ * Component for main menu.
+ *
+ * @component
+ * @return {object} (
+ *   <MainMenu />
+ * )
+ */
+
+export const MainMenu = () => {
+  // List of button links/icons we currently support
+  const icons = ['Dashboard', 'Experiments', 'Account', 'Logout'];
+
+  // Hard coding the color dark teal
+  const darkTeal = '#008489';
```

Comment:
Was unable to figure out how to pull '$teal-dark' from our SCSS file so I've hard coded the color code here.

Reply:
I think you need to import -> import '../../../scss/variables.scss';
Assign color: '$teal-dark';
Then delete const darkTeal = #008489';

Reply:
I will try this out! Thanks :D